src/Microsoft.Graph/Generated/BaseGraphServiceClient.cs

```diff
@@ -255,10 +254,6 @@ public class BaseGraphServiceClient : BaseRequestBuilder {
         public InvitationsRequestBuilder Invitations { get =>
             new InvitationsRequestBuilder(PathParameters, RequestAdapter);
         }
-        /// <summary>Provides operations to manage the collection of organizationalBrandingLocalization entities.</summary>
-        public LocalizationsRequestBuilder Localizations { get =>
```

Comment:
Confirmed as a fix for an erroneous entityset that should not have existed. 

https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/8410959?_a=files

cc @Ndiritu @ramsessanchez @baywet @rkodev